### PR TITLE
simplify changelog workflow

### PR DESCRIPTION
**Pull request by Nicoretti**
- Remove scriv as dev dependency
- Update changelog
- Update changelog section in developer guide
- Bump version number
- Update changelog

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,3 +1,42 @@
...
+
+    Failing tests in the SQLA compliance test suite, in regard to the turbodbc dialect
+    won't be addressed until explicitly required/requested by users.
+
+    Note: It is also very likely that turbodbc support will be dropped in future versions.
+
+✨ Added
+--------
+
+* Add websocket based dbapi2 compliant database driver
```
Comment (kaklakariada) [inline CHANGELOG.rst:18]: Do you usually add ticket or PR numbers to the changelog?

File: doc/developer_guide/developer_guide.rst [hunk 2]
```diff
@@ -148,43 +148,73 @@ Tests
...
-What, why and from whom to write a changelog you can read up on keepachangelog_, in
-this section we just want give the information on how to keep track of fragments
-which later will be used to create the changelog on the next published release.
+Changelog
+---------
 
-For the bookkeeping and generation of the changelog we use scriv_, so if you
-find the information in this section not sufficient we recommend to consult the
-scriv_ documentation.
+Currently wee keep our changelog in a single file *CHANGELOG.rst*.
```
Comment (kaklakariada) [inline doc/developer_guide/developer_guide.rst]:
```suggestion
Currently we keep our changelog in a single file *CHANGELOG.rst*.
```

File: doc/developer_guide/developer_guide.rst [hunk 3]
```diff
@@ -148,43 +148,73 @@ Tests
...
+For every relevant change an entry in the *unreleased* section of the changelog is
+added. When a release is created the *unreleased* section is turned into the
+section for the release, by replacing the section title by the version number and the release date of the release.
```
Comment (kaklakariada) [inline doc/developer_guide/developer_guide.rst]:
```suggestion
section for the release, by replacing the section title with the version number and the release date of the release.
```

File: doc/developer_guide/developer_guide.rst [hunk 4]
```diff
@@ -148,43 +148,73 @@ Tests
...
+    * Some changes ...
+
+Example of *CHANGELOG.rst* after *5.0.0* release:
+
+.. code-block:: rst
+
+    .. _changelog-unreleased:
+
+    Unreleased
+    ==================
```
Comment (kaklakariada) [inline doc/developer_guide/developer_guide.rst]:
```suggestion
    Unreleased
    ==========
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (Nicoretti) [reply CHANGELOG.rst:18]: Could be done but there is no single ticket for this one this is rather a preparation (first step) needed to implement the following [ticket](https://github.com/exasol/sqlalchemy-exasol/issues/105).

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**